landlab/components/bedrock_landslider/bedrock_landslider.py

```diff
@@ -0,0 +1,669 @@
+            "intent": "in",
+            "optional": False,
+            "units": "-",
+            "mapping": "node",
+            "doc": "Node array of proportion of flow sent to each receiver.",
+        },
+        "hill_topographic__steepest_slope": {
+            "dtype": float,
+            "intent": "in",
+            "optional": False,
+            "units": "-",
+            "mapping": "node",
+            "doc": "The steepest *downhill* slope",
+        },
+        "LS_sediment__flux": {
+            "dtype": float,
+            "intent": "out",
+            "optional": False,
+            "units": "m3/s",
+            "mapping": "node",
+            "doc": "Sediment flux originating from landslides \
+                (volume per unit time of sediment entering each node)",
+        },
+        "landslide__bed_erosion": {
+            "dtype": float,
+            "intent": "out",
+            "optional": False,
+            "units": "m",
+            "mapping": "node",
+            "doc": "erosion caused by bedrock landsliding ",
+        },
+        "landslide__deposition": {
+            "dtype": float,
+            "intent": "out",
+            "optional": False,
+            "units": "m",
+            "mapping": "node",
+            "doc": "deposition of bedrock derived sediment",
+        },
+        "landslide_sediment_point_source": {
+            "dtype": float,
+            "intent": "out",
+            "optional": False,
+            "units": "m3",
+            "mapping": "node",
+            "doc": "Landslide derived sediment, as point sources on all the \
+                critical nodes where landslides initiate, \
+                before landslide runout is calculated ",
+        },
+    }
+
+    _cite_as = """@Article{gmd-13-3863-2020,
+                  AUTHOR = {Campforts B., Shobe C.M., Steer P., Vanmaercke M., Lague D., Braun J.},
+                  TITLE = {BedrockLandslider 1.0: a hybrid landscape evolution model to simulate the impact of landslides and landslide-derived sediment on landscape evolution.},
+                  JOURNAL = {Geoscientific Model Development},
+                  VOLUME = {13},
+                  YEAR = {2020},
+                  NUMBER = {9},
+                  PAGES = {3863--3886},
+                  URL = {https://doi.org/10.5194/gmd-13-3863-2020},
+                  DOI = {10.5194/gmd-13-3863-2020}
+                  }"""
+
+    def __init__(
+        self,
+        grid,
+        angle_int_frict=1.0,
+        cohesion_eff=1e4,
+        landslides_return_time=1e5,
+        rho_r=2700,
+        grav=9.81,
+        fraction_fines_LS=0,
+        phi=0,
+        max_pixelsize_landslide=1e9,
+        seed=2021,
+        verbose_landslides=False,
+        landslides_on_boundary_nodes=True,
+        critical_sliding_nodes=None,
+        store_landslides_size=True,
+        store_landslides_volume=True,
+        store_landslides_volume_sed=False,
+        store_landslides_volume_bed=False,
```

Comment:
What do you think about removing these options? Instead, leave it up to the user to save landslide information from time step to time step and have, for example, `BedrockLandslider.landslides_size` always return the landslide sizes for the current time step.

Reply:
Not a bad suggestion. I had to rewrite some of the end-user scripts but this is more clean for sure!